vim/vscode-file-commands.vim

```diff
@@ -53,6 +53,7 @@ AlterCommand sav[eas] Saveas
 AlterCommand wa[ll] Wall
 AlterCommand q[uit] Quit
 AlterCommand wq Wq
+AlterCommand x Wq
```

Comment:
But you mapped to `Wq`. Maybe it should be mapped to `Xit`? You should create this mapping.

Reply:
Good suggestion